apps/website/src/components/events/virtual-ticket/TicketEditor.tsx

```diff
@@ -0,0 +1,411 @@
+            onClick={() => setSelectedSticker(null)}
+            onMouseEnter={() => toggleGhosts(true)}
+            onTouchStart={() => toggleGhosts(true)}
+            onMouseLeave={() => toggleGhosts(false)}
+            onTouchEnd={() => toggleGhosts(false)}
+          >
+            {children}
+          </div>
+        </div>
+
+        <div className="absolute bottom-0 left-1/2 z-40 -mb-3 flex -translate-x-1/2 gap-1 rounded-2xl bg-white p-0.5 text-sm shadow-xl sm:p-1 md:gap-2 lg:text-base">
+          <Button
+            className={`whitespace-nowrap ${
+              isDirty ? "bg-green-600 text-white" : "bg-gray-300 text-gray-900"
+            }`}
+            width="auto"
+            size="small"
+            disabled={!isDirty}
+            onClick={() => {
+              onSave();
+            }}
+          >
+            {isDirty ? (
+              "Save ticket"
+            ) : (
+              <>
+                <IconCheck className="h-6 w-6" />
+                Saved
+              </>
+            )}
+          </Button>
+
+          <Button
+            className="whitespace-nowrap"
+            width="auto"
+            size="small"
+            disabled={!selectedSticker}
+            onClick={() => {
+              if (selectedSticker) {
+                removeSticker(selectedSticker);
+                setSelectedSticker(null);
+              }
+            }}
+          >
+            <IconMinusCircle className="h-6 w-6" />
+            Remove sticker
+          </Button>
+
+          <Button
+            className="whitespace-nowrap"
+            width="auto"
+            size="small"
+            confirmationMessage="Are you sure you want to remove all stickers?"
+            onClick={resetStickers}
+          >
+            <IconTrash className="h-6 w-6" />
+            Reset
+          </Button>
+        </div>
+      </div>
+
+      <div className="mx-auto mt-8 flex max-w-[899px] flex-col gap-2 md:mt-12 lg:flex-row lg:gap-4">
+        {typeSafeObjectKeys(stickerPack.groups).map((groupId) => {
+          const group = stickerPack.groups[groupId]!;
+
+          return (
+            <div key={groupId} className="flex-1">
+              <div className="flex flex-row justify-center gap-1 text-sm leading-tight md:justify-start lg:flex-col lg:gap-0 lg:text-base">
+                <h3 className="font-bold lg:text-lg">{group.name}</h3>
+                <p className="text-gray-800">
+                  {group.attribution && group.attributionLink ? (
+                    <Link external href={group.attributionLink}>
+                      {group.attribution}
+                    </Link>
+                  ) : (
+                    group.attribution || <>&nbsp;</>
+                  )}
+                </p>
+              </div>
+
+              <div className="w-full overflow-auto md:overflow-hidden">
+                <ul className="flex flex-row items-center gap-1 py-2 md:flex-wrap md:gap-2 lg:gap-3">
```

Comment:
I think these should probably wrap on mobile too, rather than using scroll -- I'm not sure the fact you can scroll for more is very obvious on mobile